config/puma.rb

```diff
@@ -1,7 +1,8 @@
 persistent_timeout ENV.fetch('PERSISTENT_TIMEOUT') { 20 }.to_i
 
-threads_count = ENV.fetch('MAX_THREADS') { 5 }.to_i
-threads threads_count, threads_count
+min_threads_count = ENV.fetch('MIN_THREADS') { 5 }.to_i
```

Comment:
This changes behaviour, perhaps fallback to `MAX_THREADS` value instead? Unless changing the behaviour is a performance improvement of some sort.

Reply:
You're right, good catch. Fixed it to default to `max_thread_count` as you suggested. This should ensure identical behavior if `MIN_THREADS` is not set.